# ROS2 interface for DBoW2 FeatureVector.h

uint32 node_id
uint32[] features

